libraries/classes/Gis/GisGeometry.php

```diff
@@ -336,25 +333,21 @@ protected function getLineArrayForOpenLayers(
         $srid,
         $is_line_string = true
     ) {
-        $ol_array = 'new Array(';
+        $ol_array = 'var arr = [];';
         foreach ($lines as $line) {
+            $ol_array .= 'var lineArr = [];';
             $points_arr = $this->extractPoints($line, null);
-            $ol_array .= $this->getLineForOpenLayers(
+            $ol_array .= 'var line = ' . $this->getLineForOpenLayers(
                 $points_arr,
                 $srid,
                 $is_line_string
-            );
-            $ol_array .= ', ';
+            ) . ';';
+            $ol_array .= 'var coord = line.getCoordinates();';
+            $ol_array .= 'coord.forEach(item => lineArr.push(item));';
```

Comment:
```suggestion
            $ol_array .= 'coord.forEach(item => lineArr.push(item));';
```

Oops you can not use such features..
https://caniuse.com/#search=forEach
https://caniuse.com/#feat=arrow-functions